container-search/src/main/java/com/yahoo/prelude/statistics/StatisticsSearcher.java

```diff
@@ -77,6 +77,9 @@
     private final Value peakQPS; // peak 1s QPS
     private final Counter emptyResults; // number of results containing no concrete hits
     private final Value hitsPerQuery; // mean number of hits per query
+    private final Value hitsPerQueryBuckets;
```

Comment:
the two new "*Buckets" values doesn't do anything useful here, you can just remove them.

Reply:
Done. Doesn't this mean that also the `queryLatencyBuckets` value is redundant?

Reply:
queryLatency logs to "mean_query_latency" while queryLatencyBuckets is the one that logs to "query_latency".

actually, if you just want the new metrics in /state/v1/metrics (and so on) you should not need new Values at all; but they have the benefit of outputting their metric to vespa.log periodically which may be useful.

Reply:
Thanks for clarifying! There were several points I had missed here.